apicast/src/configuration_store.lua

```diff
@@ -16,9 +15,18 @@ local mt = { __index = _M }
 
 function _M.new()
   return setmetatable({
+    -- services hashed by id, example: {
+    --   ["16"] = service1
+    -- }
```

Comment:
Yep. Wanted to do it just for sake of having it written when I was changing the data structure :)
Will try to do that more in complicated cases.